docs/about/05_archetype.adoc

```diff
@@ -0,0 +1,64 @@
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Helidon Application Bootstrapping
+:description: Helidon Application Bootstrapping
+:keywords: helidon archetypes
+
+== Introduction
+
+<<./05_cli.adoc, Helidon CLI>> provides a convenient way to bootstrap Helidon applications. It allows you to choose from a set of archetypes i.e. application with pre-defined feature sets and lets you customize it by providing a host of options.
+
+== Archetypes
+
+Helidon provides the following set of archetypes to bootstrap your application development journey.
+
+=== Bare
+This option creates a minimal Helidon project, suitable to start from scratch. It includes only required dependencies e.g. in case of Helidon MP, it uses `helidon-microprofile-core` bundle instead of `helidon-microprofile` bundle and adds required dependencies to it. It also doesn't have out-of-box observability setup.
+
+=== QuickStart
+This option builds on `Bare` to include multiple REST operations along with default observability setup and a set of additional dependencies to enable ease of development e.g. in case of Helidon MP, it uses `helidon-microprofile` bundle instead of `helidon-microprofile-core` bundle.
+
+=== Database
+This option builds on `QuickStart` to demonstrate how to integrate with database (in-memory H2, by default). In case of, Helidon SE that uses the DbClient API while for Helidon MP that uses JPA.
+
+=== OCI
+This option builds on `QuickStart` to demonstrate integration with Oracle Cloud Infrastructure (OCI) services using the OCI SDK. Generated project showcases OpenApi-driven development approach where the practice of designing and building APIs is done first,
+then creating the rest of an application around them is implemented next. This is available for Helidon MP only.
+
+== Generated Application Structure
+
+User can scaffold a new Maven project based on these archetypes. See <<./05_cli.adoc, Helidon CLI>> and our <<se/guides/02_quickstart.adoc, Helidon SE QuickStart Guide>> or <<mp/guides/02_quickstart.adoc, Helidon MP QuickStart Guide>> for details on usage.
+
+Once the archetype is selected the other options have defaults and the project is generated in a directory named after the `artifactId` value. It mainly contains the following:
+
+- Maven structure
+- skeletal application code
+- associated unit test code
+- example Dockerfile files
+- application configuration file(s)
+- instructions to build and run application/test
+
+== Using Generated Application
+
+The easiest way to get started is follow instructions in README file and familiarize with layout and features provided to build upon them.
+
+Also look at the pom.xml. You will find the suitable Helidon parent pom, allowing you to omit the version of the different Helidon dependencies. The parent pom also brings in helidon-maven-plugin responsible for packaging of the application and providing the development mode.
```

Comment:
Looks like an attempt to explain the application parent poms. This would require more content than these few lines. I would remove the last two paragraphs.

The `helidon-maven-plugin` provides some packaging related things, however the main JAR is built with the normal Maven tooling.

Reply:
Removed two paragraph.